src/counts.jl

```diff
@@ -9,53 +9,90 @@
 typealias IntUnitRange{T<:Integer} UnitRange{T}
 
 #### functions for counting a single list of integers (1D)
+"""
+    addcounts!(r, x, levels[, wv])
 
+Fill the array `r` with the number of occurrences in `x` of
```

Comment:
Are you sure? Reading the code for `addcounts!` it looks to me like `r` is getting overwritten rather than added to. Maybe I'm not interpreting the code correctly.